docs/endpoints/post-identity-map.md

```diff
@@ -34,7 +34,7 @@ You must encrypt all requests using your secret. For details, and code examples
 
 | Path Parameter | Data Type | Attribute | Description |
 | :--- | :--- | :--- | :--- |
-| `{environment}` | string | Required | Integration environment: `https://operator-integ.uidapi.com`<br/>Production environment: `https://prod.uidapi.com`<br/>For a full list, including regional operators, see [Environments](../getting-started/gs-environments.md). |
+| `{environment}` | string | Required | Integration environment: `https://operator-integ.uidapi.com`<br/>Production environment: `https://prod.uidapi.com`<br/>For a full list of valid base URLs, including regional operators, see [Environments](../getting-started/gs-environments.md). |
```

Comment:
We do highlight that prod and integ are different in https://unifiedid.com/docs/getting-started/gs-environments
Do we really need to duplicate that information everywhere that we talk about URLs?

> But in this scenario is [prod.uidapi.com](https://prod.uidapi.com/) not a default because for the API there is no default, I guess that's what you're saying?
You're correct that direct API callers don't have any default. But I was referring to the fact that it's not appropriate to say:
"Production environment: https://prod.uidapi.com" in this document. Users will be misled to think that if they want production, that's the URL they should put. And that's not true, eg here's a quote from @atarassov-ttd :

> You would not want to hit [prod.uidapi.com](http://prod.uidapi.com/) for all advertisers:
> This increases latency
> Some advertisers may want to control which regions emails/phones are kept within, e.g. to comply with upcoming (?) Australia data protection laws some organisations want to make sure emails are never sent outside of Australia.

Reply:
@jon8787 understood and thank you. Additional updates. Over to you.